SpotlightDownloader/Program.cs

```diff
@@ -148,7 +152,7 @@ static void Main(string[] args)
                                     }
                                     catch (Exception e)
                                     {
-                                        Console.Error.WriteLine(e.GetType() + ": " + e.Message);
+                                        Console.Error.WriteLine("Failed to set wallpaper: " + e.Message);
```

Comment:
```diff
-                                        Console.Error.WriteLine(e.GetType() + ": " + e.Message);
+                                        Console.Error.WriteLine("Failed to set wallpaper: " + e.Message);
```

Exception type should be kept in error message, because the exception message itself is sometimes unhelpful.